config.json

```diff
@@ -41,9 +41,9 @@
     "Firefox ESR"
   ],
   "settings": {
-    "homepage_new_products_count": 12,
-    "homepage_featured_products_count": 8,
-    "homepage_top_products_count": 8,
+    "homepage_new_products_count": 5,
+    "homepage_featured_products_count": 4,
+    "homepage_top_products_count": 4,
```

Comment:
Updating the new/featured/popular products counts in tests. There'll be another PR for that.

Reply:
Tracking test updates in https://github.com/bigcommerce/bigcommerce/pull/24622.